.gitignore

```diff
@@ -42,3 +42,5 @@ testem.log
 Thumbs.db
 src/app/credentials/youtubekey.model.ts
 src/app/credentials/vimeokey.model.ts
+src/app/core/credentials/vimeokey.model.ts
+src/app/core/credentials/youtubekey.model.ts
```

Comment:
imo nazewnictwo jest błędne, bo w tym pliku nie mamy modelu a tylko klucz i w zasadzie to nie ma potrzeby tworzenia pliku z kluczem per ap, wrzuć to do jednego pliku o nazwie credentials i tyle no i nie rozumiem czemu raz masz 
src/app/credentials a za drugim razem src/app/core/credentials